.github/workflows/java-sample-app-ecr-deploy.yml

```diff
@@ -9,8 +9,12 @@ permissions:
   id-token: write
   contents: read
 
+env:
+  E2E_TEST_ACCOUNT_ID: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ACCOUNT_ID }}
+  E2E_TEST_ROLE_NAME: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ROLE_NAME }}
+
 jobs:
-  upload-main-service-image:
+  upload-java-main-service-image:
```

Comment:
Can we shorten this actually? Last I checked it was hard to read what region a job was running because of how long this is.

Reply:
Shortened